sgpt/handlers/repl_handler.py

```diff
@@ -34,6 +48,8 @@ def handle(self, prompt: str, **kwargs: Any) -> None:  # type: ignore
         while True:
             # Infinite loop until user exits with Ctrl+C.
             prompt = typer.prompt(">>>", prompt_suffix=" ")
+            if prompt == "{":
```

Comment:
I think we can use Python like multi-line text syntax with triple quotes `"""` (which is well known):
```python
>>> """
... test
... text
... """
-> 'test\ntext'
```
So in this case `"""` triggers multi-line input (start/end) and `...` indicates we are currently in multi-line mode.